client/src/app/site/motions/modules/motion-detail/components/motion-title-change-recommendation/motion-title-change-recommendation.component.scss

```diff
@@ -0,0 +1,11 @@
+::ng-deep {
+    h3 {
+        margin-top: 0.3em;
+        margin-bottom: 0.3em;
+        font-size: 1em;
+    }
+
+    mat-form-field {
```

Comment:
can probably be accessed without shadow piercing if you set View Encapsulation to none.

```scss
.motion-content {
    .mar-form-field {
        width: 100%;
    }
}